toplevel/opttopmain.ml

```diff
@@ -183,7 +183,6 @@ module Options = Main_args.Make_opttop_options (struct
   let _labels = clear classic
   let _alias_deps = clear transparent_modules
   let _no_alias_deps = set transparent_modules
-  let _dlinscan = set use_linscan
```

Comment:
Naively I would expect the native toplevel to call a register allocator, so the linscan option would be used at some point. Why is it "not used" by this code? Is the linscan setting made somewhere else that affects opttopmain, or is this option never available for code produced by opttopmain?

Reply:
It could be used. However, the functor that builds the arguments for the native toplevel takes a 
Opttop_options as argument, wich itself derives its native compilation option from the `Optcommon_options` module type. And this module type does not include a `_dlinscan` or `_linscan` hook (see https://github.com/ocaml/ocaml/blob/trunk/driver/main_args.mli#L160 ).

Reply:
So shouldn't we move some options from Optcomp to Optcommon when it makes sense? I would think of `linscan`, `no_float_const_prop`, `dinterval`, and maybe (I'm not sure) `S`?

Reply:
It is clearly another possibility. but I would rather do it in a subsequent PR , once it become easier to move options from one option group to another.